packages/block-editor/src/components/url-popover/test/__snapshots__/index.js.snap

```diff
@@ -5,7 +5,7 @@ exports[`URLPopover matches the snapshot in its default state 1`] = `
   <span>
     <div
       class="components-popover block-editor-url-popover is-positioned"
-      style="position: absolute; top: 0px; left: 0px; transform: none;"
+      style="position: absolute; top: 0px; left: 0px; transform: translateZ(0);"
```

Comment:
I'm guessing this is an internal implementation details to framer motion animations change. Maybe perf related

Reply:
seems like it